packages/qdrant-loader-core/src/qdrant_loader_core/graph/__init__.py

```
@@ -0,0 +1,59 @@
+
+    if _graph_store is None:
+        async with _graph_store_lock:
+            if _graph_store is None:
+                # Config from settings
+                if get_settings is not None:
+                    try:
+                        settings = get_settings()
+                        graph_cfg = getattr(settings.global_config, "graph", None)
+
+                        if graph_cfg is not None:
+                            host = host or graph_cfg.connection.host
```

Ruff (0.15.15)

[error] Undefined name `host`

(F821)